clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -693,14 +693,13 @@ ItaniumMangleContextImpl::getEffectiveDeclContext(const Decl *D) {
     if (VD->isExternC())
       return getASTContext().getTranslationUnitDecl();
 
-  if (const auto *FD = D->getAsFunction()) {
-    if (FD->isExternC())
```

Comment:
why removing the `isExternC()`? this will change the behavior.

Reply:
It's back.